src/Host/Client/Impl/Host/LocalBrokerClient.cs

```diff
@@ -100,7 +100,7 @@ public LocalBrokerClient(string name, string rHome, string rhostDirectory = null
                     process = Process.Start(psi);
                     process.EnableRaisingEvents = true;
 
-                    var cts = new CancellationTokenSource(10000);
+                    var cts = new CancellationTokenSource(100000);
```

Comment:
This is an awfully long timeout for a local broker. Do we really want to wait for almost 2 minutes before reporting failure to start?